### ISSUE TITLE
Bot should avoid drowning

### ISSUE THREAD

**Issue by DominicEliot** [OreaDev/libBetterBot#20]
If the bot ends combat swimming underwater and it decides it needs to drink/eat it tries to eat but it can't. Then it drowns 😞. It should try and surface when low breath.

**Comment by OreaDev** [comment]
Yeah, this is a big problem currently. Going to make this high priority
